widget/widget_date_input.go

```diff
@@ -0,0 +1,195 @@
+package widget
+
+import (
+	"fmt"
+	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/widget"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const ConstDefaultDateFormat = "02-Jan-2006"
+
+var DayPos = []int{0, 2}
```

Comment:
Should these fields be part of the public API?

Reply:
No